resources/serverless/templates/runtimes.yaml

```diff
@@ -73,3 +73,34 @@ data:
     RUN rm -rf /etc/pip.conf
 
     USER 1000
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: dockerfile-java17-jvm-alpha
+  namespace: kyma-system
+  labels:
+    serverless.kyma-project.io/config: runtime
+    serverless.kyma-project.io/runtime: java17-jvm-alpha
+data:
+  Dockerfile: |-
+    FROM {{ include "imageurl" (dict "reg" .Values.global.containerRegistry "img" .Values.global.images.function_runtime_java17_jvm_alpha) }} as builder
+
+    ARG BUILD_DIR=/build
+    ARG SOURCE_DIR=/src
+    ARG DEPS_DIR=/src
```

Comment:
Do we need both? The have the same value are used once each.

Reply:
Unfortunately yes. I've done it to be able to use the same image for local build and kaniko build.

Reply:
Probably a good idea to add a comment about that to avoid "fixing it" in the future ;)

Reply:
Added :)